app/models/task_model.py

```diff
@@ -0,0 +1,22 @@
+from app import db
+
+class Task(db.Model):
+    task_id = db.Column(db.Integer, primary_key=True)
+    title = db.Column(db.String)
+    description = db.Column(db.String)
+    completed_at = db.Column(db.DateTime, default=None)
+    goal_id = db.Column(db.Integer, db.ForeignKey('goal.goal_id'), nullable=True)
+    goal = db.relationship("Goal", back_populates="tasks")
+    
```

Comment:
👍 Nice work setting up the `Task` model. The comment from the `Goal` model about setting columns to `nullable=False` can also be applied here.